templates/header.html

```diff
@@ -10,7 +10,11 @@
         </a>
   
         <ul class="nav nav-pills">
+          {% if not current_user.is_authenticated %}
```

Comment:
it would be more readable to start with `if` instead of `if not`:

`{% if current_user.is_authenticated %}`